app/components/TabBar/tabBar.css

```diff
@@ -3,14 +3,28 @@
 {
     flex: 0;
     height: 100%;
-    padding-left:     10px;
-    padding-right:     10px;
+    padding-left: 10px;
+    padding-right: 10px;
     background: rgba( 207, 208, 209, 1);
     z-index: 4;
     font-size: 1.4rem;
     -webkit-app-region: drag;
 }
 
+.containerMac
+{
+    flex: 0;
```

Comment:
Here, I _think_ the only thing needed is `padding-left: 100px`, the rest should not need to be defined if this is just an extra class added TO the current tabBar. (no top padding is needed to be honest).